<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>地图</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="lstMap">
        <property name="currentRow">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>BicycleRiding</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BicycleBalancing</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>算法</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="lstAgent">
        <property name="currentRow">
         <number>0</number>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>QLearning</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sarsa</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfigAgent">
        <property name="text">
         <string>参数设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>表示</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="lstRepresentation">
        <property name="currentRow">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>KernelizediFDD</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfigRepresentation">
        <property name="text">
         <string>参数设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>策略</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QListWidget" name="lstPolicy">
        <property name="currentRow">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>eGreedy</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfigPolicy">
        <property name="text">
         <string>参数设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
